Makefile

```diff
@@ -0,0 +1,26 @@
+all: build
+.PHONY: all
+
+GO=GO111MODULE=on GOFLAGS=-mod=vendor go
+GO_BUILD_BINDIR := bin
+
+# Include the library makefile
+include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machinery/make/, \
+	golang.mk \
+	targets/openshift/images.mk \
+)
+
+# generate image targets
+IMAGE_REGISTRY ?=registry.svc.ci.openshift.org
+IMAGE_TAG ?=dev
+$(call build-image,cluster-resource-override-admission,$(IMAGE_REGISTRY)/autoscaling/cluster-resource-override,./images/ci/Dockerfile,.)
+
+# exclude e2e test from unit tests
+GO_TEST_PACKAGES :=./pkg/... ./cmd/...
+
+# re-use test-unit target for e2e tests until we
+.PHONY: test-e2e
+test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
+test-e2e: test-unit
```

Comment:
Remove this as you have no e2e tests. Typically, we add the e2e tests the operators.

Reply:
I have a couple of e2e tests that validates the `ClusterResourceOverride` admission webhook. I would like e2e to be part of the CI to ensure that the admission webhook is not broken due to changes made to it.

Reply:
The best we can do right now is to add it to the operator. In the future maybe we can get the operator e2e to run as well with a change on this repo.

Reply:
> In the future maybe we can get the operator e2e to run as well with a change on this repo.
I like this idea, I am taking this as an action item to find out how we can do this in our CI.